src/blocks/media-block/image-block/SimpleImageBlock.test.tsx

```
@@ -0,0 +1,342 @@
+    // Restore original implementations
+    window.IntersectionObserver = originalIntersectionObserver;
+    window.Image = originalImage;
+  });
+  
+  beforeEach(() => {
+    // Clear all mocks before each test
+    const vi = (global as unknown as GlobalWithVi).vi || (global as unknown as GlobalWithVi).jest;
+    vi.clearAllMocks();
+    
+    // Create a function that matches the expected signature
+    const mockImplementation = function(this: HTMLImageElement | void, width?: number, height?: number): HTMLImageElement {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)